lark/utils.py

```diff
@@ -4,7 +4,7 @@
 from collections import deque
 
 ###{standalone
-import sys, re
```

Comment:
Did you check that this is truly unused, even when a standalone file is generated?

Reply:
I ran the tests and assumed it checked for it. But I just did a test myself, and it is indeed needed. Reverting.